src/awfy.js

```diff
@@ -1296,9 +1298,50 @@ export const BENCHMARKS = {
         frameworkId: RAPTOR_FRAMEWORK_ID,
         suite: 'raptor-speedometer-geckoview',
       },
+      fenix: {
+        color: COLORS.fenix,
+        label: 'Fenix',
+        frameworkId: BROWSERTIME_FRAMEWORK_ID,
+        suite: 'speedometer',
+        option: 'pgo',
+        extraOptions: ['fenix'],
+      },
     },
     label: 'Speedometer',
```

Comment:
adding `yLabel: 'Score',` should manually set the unit label correctly

it puts ms by default (if lower_is_better is false) https://github.com/mozilla-frontend-infra/firefox-performance-dashboards/blob/master/src/utils/chartJs.js#L40

Reply:
As this is a pre-existing issue, and it affects a number of other graphs I'd prefer to fix it separately. Could you file an issue to use the unit provided by the Perfherder data in all cases?

Reply:
hmm.. the code you reference suggests that score should be used if lower is better...

Reply:
It may be that it doesn't get it properly. Maybe it tries to take the lower_is_better from a subtest of speedometer. I think that on health dashboard this is solved with filtering by missing the test property (only the subtests have the test property)

```
"bb2d328ea20b53e03cec65d3c3a7718cf23f5053": {
	"id": 2341492,
	"framework_id": 13,
	"option_collection_hash": "f69e1b00908837bf0550250abb1645014317e8ec",
	"machine_platform": "android-hw-g5-7-0-arm7-api-16",
	"suite": "speedometer",
	"test": "jQuery-TodoMVC/Adding100Items/Async",
	"application": "fenix",
	"parent_signature": "6e726ab9c9f61b85005e2a97f92704b3b9787b59",
	"extra_options": [
		"fenix"
	],
	"measurement_unit": "ms"
},
"bb2da9dd6099ed06df3a7c29930b7ee1c73c53c2": {
	"id": 2340502,
	"framework_id": 13,
	"option_collection_hash": "102210fe594ee9b33d82058545b1ed14f4c8206e",
	"machine_platform": "android-hw-g5-7-0-arm7-api-16",
	"suite": "speedometer",
	"lower_is_better": false,
	"application": "fenix",
	"has_subtests": true,
	"extra_options": [
		"fenix"
	],
	"measurement_unit": "score"
},
```

Reply:
I created this separate issue for the measurement unit problem: https://github.com/mozilla-frontend-infra/firefox-performance-dashboards/issues/292